ONSdigital/eq-survey-runner
# Improved test coverage of converter.py

### What is the context of this PR?
#1466 removed a schema-specific test. In an effort to make sure we don't lose any test coverage, more convert tests have been added.

- Added test conversion for the following answer types:
   - radio
   - number
   - percentage
   - textarea
   - currency
   - dropdown
   - date
   - unit
   - relationship

### How to review 
1. Run `pipenv run scripts/run_tests_unit.sh`
2. Ensure that all units tests pass.
3. Ensure all the new tests are correct and the structure is valid.
